android-agent/src/main/kotlin/io/opentelemetry/android/agent/dsl/EndpointConfiguration.kt

```diff
@@ -7,6 +7,6 @@ package io.opentelemetry.android.agent.dsl
 
 @OpenTelemetryDslMarker
 class EndpointConfiguration internal constructor(
-    var url: String,
+    var url: String?,
```

Comment:
can we make this non nullable? feels wrong allowning a nullable url for an endpoint configuration